lib/softoken/pkcs11c.c

```diff
@@ -8279,6 +8279,54 @@ NSC_DeriveKey(CK_SESSION_HANDLE hSession,
                                   sourceKey, key);
             break;
 
+        case CKM_SP800_108_COUNTER_KDF:
+            if (BAD_PARAM_CAST(pMechanism, sizeof(CK_SP800_108_KDF_PARAMS))) {
+                crv = CKR_MECHANISM_PARAM_INVALID;
+                break;
+            }
+            if (sftk_isTrue(key, CKA_TOKEN)) {
```

Comment:
Should I move these two checks (`sftk_isTrue` and `sftk_DeriveSensitiveCheck`) into `kbkdf_Counter`?